checks/remotesettings/clients_error_rate.py

```diff
@@ -0,0 +1,125 @@
+"""
+The percentage of reported errors in Uptake Telemetry should be under the specified
+maximum.
+
+For each collection whose error rate is above the maximum, the total number of events
+for each status is returned. The min/max timestamps give the datetime range of the
+dataset obtained from https://sql.telemetry.mozilla.org/queries/64808/
+"""
+import os
+from collections import defaultdict
+from typing import Dict, List
+
+import aiohttp
+
+from poucave.typings import CheckResult
+
+
+EXPOSED_PARAMETERS = ["max_percentage", "min_total_events"]
+
+REDASH_URI = (
+    f"https://sql.telemetry.mozilla.org/api/queries/64808/results.json?api_key="
```

Comment:
I thought Redash was the way to go (JSON, API Key, periodic schedule...).

You can review the query, you're allowed (and most of our team mates) to view/edit it.

I can investigate about BigQuery, but for what I can see it will be more painful in terms of authentication, etc.

Reply:
We spoke online about this. The query takes minutes to run, which makes running it directly from a request problematic. Using Redash allows us to schedule the query so that when we make requests, we get results immediately. On the other hand, it fragments the code for this service, and it introduces a weird kind of bus factor since the query is under Mat's account. We agreed that using Redash was fine for now but we would introduce an issue to follow up and see if there's a better way to address this (#60).